_includes/perf_example.html

```diff
@@ -1,8 +1,8 @@
-<div class="usa-alert usa-alert-info">
+<div class="usa-alert usa-alert-info usa-alert-paragraph">
   <div class="usa-alert-body">
-    <h3 class="usa-alert-heading">example</h3>
+    <h3 class="usa-alert-heading">Example</h3>
```

Comment:
Yeah, for now at least. We should circle back and make a better version of the alerts at a later date.